client-participation/.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 8.17.0
```

Comment:
Inline comment to explain what this is, so folks know why it's there?

Reply:
since asdf-nodejs is probably a lot less common, I replaced these files with .nvmrc files. Most JS devs use or at least are familiar with nvm, and there are other references to nvm in this repo.
Also asdf-nodejs (my version manager of choise) can be made to use .nvmrc files.